Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Build the manager binary
+FROM golang:1.13 as builder
+WORKDIR /workspace
+# Copy the jsonnet source
+COPY environments/operator/ environments/operator/
+COPY components/ components/
+COPY jsonnetfile.json jsonnetfile.json
+# Build
+RUN GO111MODULE="on" go get github.com/jsonnet-bundler/jsonnet-bundler/cmd/jb
+RUN jb install
```

Comment:
cpaas builds are offline only, so for the downstream fork we will need to check in the `vendor/` directory from jsonnet as well, which reminds me that we should try to split the go and jsonnet vendor directories

Reply:
Should this be done in this PR or a follow-up one?

Reply:
not in this repo at all I think, let's take care of this downstream